llvm/test/CodeGen/PowerPC/coalescer-remat-with-undef-implicit-def-operand.mir

```diff
@@ -0,0 +1,33 @@
+# RUN: llc -mtriple=powerpc64le-unknown-linux-gnu -verify-coalescing -run-pass=register-coalescer \
+# RUN:   -o - %s | FileCheck %s
+---
+name:            _Z13testTransposeIfLj31ELj17EEvv
+alignment:       16
+tracksRegLiveness: true
+frameInfo:
+  maxAlignment:    128
+stack:
+  - { id: 0, size: 128, alignment: 128, local-offset: -128 }
+  - { id: 1, size: 4096, alignment: 16, local-offset: -4224 }
```

Comment:
Nit (feel free to ignore): there don't seem to be any stack accesses here. Do we need these?